arch/arm64/Kconfig

```diff
@@ -261,12 +251,34 @@ config ARCH_CHIP
 	default "imx8"      if ARCH_CHIP_IMX8
 	default "imx9"      if ARCH_CHIP_IMX9
 
-config ARM_HAVE_NEON
+config ARM64_HAVE_PSCI
+	bool "ARM PCSI (Power State Coordination Interface) Support"
+	default n
+	---help---
+		This Power State Coordination Interface (PSCI) defines
+		a standard interface for power management. the PCSI need
+		to implement handling firmware at EL2 or EL3. The option
+		maybe not applicable for arm core without PCSI firmware
+		interface implement
+
+config ARM64_PSCI
+	bool "Enabled PSCI"
+	depends on ARM64_HAVE_PSCI
+	default n
```

Comment:
Should we change this to `default y` to minimize the impact on existing boards?

Reply:
it's fine with either

Reply:
I have changed to `default y` so existing `defconfig` still works by default.